### PR TITLE
Replace django-db-locking with Redis locks

### PR DESCRIPTION
**Pull request by nielsvanoch**

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Django>=2.1.5,<2.3
 celery>=4.0.0
 python-dateutil
-django-db-locking>=2.0.4
+redis
```
Comment (Timvde) [inline requirements/requirements.txt]: No version?

File: djtriggers/models.py [hunk 2]
```diff
@@ -103,7 +103,8 @@ def process(self, force=False, logger=None, dictionary=None, use_statsd=False):
         # The lock assures no two tasks can process a trigger simultaneously.
         # The check for date_processed assures a trigger is not executed multiple times.
         try:
-            with NonBlockingLock.objects.acquire_lock(self):
+            with Lock(redis=Redis.from_url(settings.CELERY_BROKER_URL), name='djtriggers-' + str(self.id),
```
Comment (Timvde) [inline djtriggers/models.py]: Why do you use `CELERY_BROKER_URL` for this? I'm okay with Django's cache, but this seems like a stretch.

File: requirements/requirements.txt (hunk 1 above)
Comment (nielsvanoch) [reply requirements/requirements.txt]: Pinned it to `>=3.0.0`.

File: djtriggers/models.py (hunk 2 above)
Comment (nielsvanoch) [reply djtriggers/models.py]: Introduced a new setting for it.

**Pull request merged**